src/main/java/pl/touk/sputnik/processor/spotbugs/SpotBugsProcessor.java

```diff
@@ -121,4 +130,23 @@ private String getExcludeFilterFilename() {
         log.info("Using SpotBugs exclude filter file {}", excludeFilterFilename);
         return excludeFilterFilename;
     }
+
+    private void loadAllSpotbugsPlugins() {
+        String pluginsLocation = config.getProperty(GeneralOption.SPOTBUGS_PLUGINS_LOCATION);
+        if (pluginsLocation != null) {
```

Comment:
Please use StringUtils here so we can be safe with null and empty strings too. Also you can revert your if condition and use example below to achieve less idents.

```
if (StringUtils.isBlank(pluginsLocation)) then {
    return
}
```